templates/dataset.html

```diff
@@ -586,7 +599,7 @@ <h3>Tasks</h3>
                                           <p class="help-block">Generate files containing the tweet ids of the dataset.</p>
                                       </label>
                                     </div>
-                                    <button type="submit" form="datasetExportsForm" name="generate_tasks" value="true" class="btn btn-primary" {% if task_id %} disabled="true" {% endif %}>Generate</button>
+                                    <button type="submit" id="datasetExportsId" form="datasetExportsForm" name="generate_tasks" value="true" class="btn btn-primary" {% if task_id %} disabled="true" {% endif %}>Generate</button>
```

Comment:
Since this is a required field, recommend adding an indication for screen-readers at a minimum. See https://webaim.org/techniques/forms/controls#requiredfields